File: Codigo/Programación_de_computadores/Tareas/Comparaciones/ordenamiento_de_valores/ordenamiento_de_valores.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main(){
    int x;
    int y;
    int z;
    cout<<"Ingresa tu primer valor: ";
    cin>>x;
    cout<<"Ingresa tu segundo valor: ";
    cin>>y;
    cout<<"Ingresa tu tercer valor: ";
    cin>>z;
    if (x>=y && x>=z){
        cout<<"Tu mayor valor es: "<<x;
    }
    else if (y>=x && y>=z){
        cout<<"Tu mayor valor es: "<<y;
    }
    else {
        cout<<"Tu mayor valor es: "<<z;
    }
}

File: Codigo/Programación_de_computadores/Tareas/Comparaciones/ordenamiento_de_valores/ordenamiento_de_valores.py
x = int(input("Ingresa tu primer valor: "))
y = int(input("Ingresa tu segundo valor: "))
z = int(input("Ingresa tu tercer valor: "))

if x >= y and y >= z:
    print("Descendentemente hacia abajo");print(x);print(y);print(z)
elif x >= z and z >= y:
    print("Descendentemente hacia abajo");print(x);print(z);print(y)
elif y >= x and x >= z:
    print("Descendentemente hacia abajo");print(y);print(x);print(z)
elif y >= z and z >= x:
    print("Descendentemente hacia abajo");print(y);print(z);print(x)
elif z >= x and x >= y:
    print("Descendentemente hacia abajo");print(z);print(x);print(y)
elif z >= y and y >= x:
    print("Descendentemente hacia abajo");print(z);print(y);print(x)
